GCGS-Central-Digital-Platform.sln

```diff
@@ -86,6 +86,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CO.CDP.Authentication", "Li
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CO.CDP.Authentication.Tests", "Libraries\CO.CDP.Authentication.Tests\CO.CDP.Authentication.Tests.csproj", "{C882BBF8-B8EF-4FF4-95E4-159DBA9F87E7}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CO.CDP.Forms.WebApiClient.Tests", "Libraries\CO.CDP.Forms.WebApiClient.Tests\CO.CDP.Forms.WebApiClient.Tests.csproj", "{10DA49F2-1723-44DC-AC9D-78C8C557653B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CO.CDP.Forms.WebApiClient", "Libraries\CO.CDP.Forms.WebApiClient\CO.CDP.Forms.WebApiClient.csproj", "{EB6B7ADD-13BD-47C4-9391-96AAC33AF51D}"
+	ProjectSection(ProjectDependencies) = postProject
+		{4298A4EF-B336-4809-95F0-AF5D73266A18} = {4298A4EF-B336-4809-95F0-AF5D73266A18}
+	EndProjectSection
```

Comment:
@JBaigGoaco this is critical as Forms.WebApi must be built before Forms.WebApiClient. Otherwise the contract json file won't be available to the WebApiClient.

Reply:
Great. Thank s a lot